.gitignore

```diff
@@ -3,4 +3,6 @@
 # directories
 .test/
 
-.DS_Store
```

Comment:
I never experienced problems by not pushing it to git. It will always be generated locally if needed, so I would just leave it out, since I never saw a `.DS_Store` file in a git repo.

Reply:
I meant it the other way around :) it should be in .gitignore and therefore not be pushed to git. The commit seems to remove it from .gitignore.

Reply:
Oh, I see. Will fix that xD

Reply:
It was still in the .gitignore, I just moved it to the top.